lms/djangoapps/instructor/views/api.py

```diff
@@ -2584,6 +2585,57 @@ def spoc_gradebook(request, course_id):
 
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+def get_blank_lti(request, course_id):  # pylint: disable=W0613
+    """
+    Respond with 4-column CSV output of ID, email, grade (blank), max_grade (blank), and comments (blank)
+    """
+    course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
@stvstnfrd we should be using the new course key function right? I don't remember the call